portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,214 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Jilene's Portfolio</title>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css"> -->
```

Comment:
What do you think about either deleting these or commenting as to why they're commented out?

Reply:
I'm glad you pointed that out. I was a bit confused as to which font types to include for different browser support. After further research, it seems as though most modern browsers support WOFF2, but IE11 only supports WOFF. I'll remove the .eot and .svg files for Roboto as they seem obsolete.

Reply:
I originally commented out those links because I planned on implementing the Animate on scroll library. Would it be best practice to remove the lines all together until I fully implement that feature?

Reply:
Today I learned -- that makes sense about the woff/woff2 files! Thanks for explaining to me.

It would be best practice to remove these comments. I think it would be ok to add a comment above like "TODO: implement animate on scroll" if you'd like to keep it!

Reply:
Gotcha, done!